src/storage/serialization/serialize_nodes.cpp

```diff
@@ -135,26 +135,26 @@ void CSVReaderOptions::Serialize(Serializer &serializer) const {
 	serializer.WritePropertyWithDefault<idx_t>(114, "buffer_size", buffer_size);
 	serializer.WriteProperty<MultiFileReaderOptions>(115, "file_options", file_options);
 	serializer.WritePropertyWithDefault<vector<bool>>(116, "force_quote", force_quote);
-	serializer.WritePropertyWithDefault<string>(117, "rejects_table_name", rejects_table_name);
+	serializer.WriteProperty<CSVOption<bool>>(117, "store_rejects", store_rejects);
```

Comment:
Could we try to minimize the breakage, and use default properties where possible?